docs/services/data/dataset.md

```diff
@@ -41,26 +41,7 @@ To interact with datasets programmatically, use the [data client API](/appendix/
 
 {{< readfile "/static/include/app/apis/generated/dataset-table.md" >}}
 
-## Sample dataset
-
-If you are testing, you can use one of two provided sample dataset, one of shapes and the other of a wooden figure:
-
-{{<imgproc src="/tutorials/data-management/shapes-dataset.png" resize="1200x" declaredimensions=true style="max-width:400px" alt="The shapes dataset." class="imgzoom fill aligncenter">}}
-
-{{< imgproc src="/tutorials/filtered-camera-module/viam-figure-dataset.png" style="max-width:400px" alt="The datasets subtab of the data tab in the Viam app, showing a custom 'viam-figure' dataset of 25 images, most containing the wooden Viam figure" class="imgzoom fill aligncenter" resize="1400x" >}}
-
-1. [Download the shapes dataset](https://storage.googleapis.com/docs-blog/dataset-shapes.zip) or [download the wooden figure dataset](https://storage.googleapis.com/docs-blog/dataset-figure.zip).
-1. Unzip the download.
-1. Open a terminal and go to the dataset folder.
-1. In it you will find a Python script to upload the data to the Viam app.
-1. Open the script and fill in the constants at the top of the file.
-1. Run the script to upload the data into a dataset in Viam app:
-
-   ```sh {class="command-line" data-prompt="$" }
-   python3 upload_data.py
-   ```
-
-## Next steps
+## How-to guide
```

Comment:
```suggestion
## CLI

To interact with datasets from the command line, use the [CLI `dataset` command](/cli/#dataset).

Example usage:

```sh {class="command-line" data-prompt="$"}
# create a new dataset
viam dataset create --org-id=123 --name=MyDataset

# list dataset information from the specified org
viam dataset list --org-id=123

# add images tagged with the "example" tag between January and October of 2023 to dataset abc
viam dataset data add filter --dataset-id=abc --location-id=123 --org-id=456 --start=2023-01-01T05:00:00.000Z --end=2023-10-01T04:00:00.000Z --tags=example
```

## How-to guide
```

Reply:
This comment seems to have rendered oddly

Reply:
In any case this seems super duplicative since we already have all this and more in the CLI page, which we already link to, and which is already part of reference docs

Reply:
It renders oddly because of the code snippet ending - that ends the suggestion because they use the same closing syntax

Reply:
Ok let's leave it as is for now. I am a bit concerned the CLI commands may not be highlighted enough but the same is true across the docs right now